src/modal/story.tsx

```diff
@@ -47,12 +52,48 @@ class ModalOpener extends Component<ModalProps> {
           </div>
         </Modal>
 
-        <Modal {...this.props} onClose={this.closeModal2} isOpen={this.state.modalOpen2}>
-          <div>
-            <h1 id="label2">Modal 2</h1>
-            <p id="description2">This is the description of my modal</p>
-            <img src="http://placekitten.com/g/216/144" width="216" height="144" alt="A kitten" />
-          </div>
+        <Modal
+          {...this.props}
+          onClose={this.closeModal2}
+          isOpen={this.state.modalOpen2}
+          noHorizontalSpacing
+          layoutModeEnabled
+        >
+          <ModalBody>
+            <ModalFog aria-hidden="true" isLoading={this.props.isLoading} />
```

Comment:
Is there a way to prevent the body from scrolling?

Reply:
Do we need to prevent the scroll? I think we discussed it but I wasn't sure and there's nothing about this in the specifications

Reply:
We discussed it with Jonas but the specs were never updated, so I'm fine with keeping it like this 👍.

Reply:
Ok I'll have a look if I can make this work in another PR.